src/SonarScanner.MSBuild.Common/Resources.Designer.cs

```diff
@@ -22,477 +22,592 @@ namespace SonarScanner.MSBuild.Common;
-    internal static string ERROR_Properties_InvalidPropertiesFile {
-        get {
+    internal static string ERROR_Properties_InvalidPropertiesFile
+    {
+        get
+        {
             return ResourceManager.GetString("ERROR_Properties_InvalidPropertiesFile", resourceCulture);
         }
     }
-    
+
     /// <summary>
     ///   Looks up a localized string similar to The settings file could not be found.
     /// </summary>
-    internal static string ERROR_SettingsFileNotFound {
-        get {
+    internal static string ERROR_SettingsFileNotFound
+    {
+        get
+        {
             return ResourceManager.GetString("ERROR_SettingsFileNotFound", resourceCulture);
         }
     }
-    
+
     /// <summary>
     ///   Looks up a localized string similar to A server certificate could not be validated. Possible cause: you are using a self-signed SSL certificate but the certificate has not been installed on the client machine. Please make sure that you can access {0} without encountering certificate errors..
     /// </summary>
-    internal static string ERROR_TrustFailure {
-        get {
+    internal static string ERROR_TrustFailure
+    {
+        get
+        {
             return ResourceManager.GetString("ERROR_TrustFailure", resourceCulture);
         }
     }
-    
+
     /// <summary>
     ///   Looks up a localized string similar to Could not authorize while connecting to the SonarQube server. Check your credentials and try again..
     /// </summary>
-    internal static string ERROR_UnauthorizedConnection {
-        get {
+    internal static string ERROR_UnauthorizedConnection
+    {
+        get
+        {
             return ResourceManager.GetString("ERROR_UnauthorizedConnection", resourceCulture);
         }
     }
-    
+
     /// <summary>
     ///   Looks up a localized string similar to The name of the SonarQube server could not be resolved. Check the url is correct and that the server is available. Url: {0}.
     /// </summary>
-    internal static string ERROR_UrlNameResolutionFailed {
-        get {
+    internal static string ERROR_UrlNameResolutionFailed
+    {
+        get
+        {
             return ResourceManager.GetString("ERROR_UrlNameResolutionFailed", resourceCulture);
         }
     }
-    
+
     /// <summary>
     ///   Looks up a localized string similar to WARNING: .
     /// </summary>
-    internal static string Logger_WarningPrefix {
-        get {
+    internal static string Logger_WarningPrefix
+    {
+        get
+        {
             return ResourceManager.GetString("Logger_WarningPrefix", resourceCulture);
         }
     }
-    
+
+    /// <summary>
+    ///   Looks up a localized string similar to File could not be added to analysis files: &apos;{0}&apos;..
+    /// </summary>
+    internal static string MSG_AnalysisFileCouldNotBeAdded
+    {
+        get
+        {
+            return ResourceManager.GetString("MSG_AnalysisFileCouldNotBeAdded", resourceCulture);
```

Comment:
This is the new addition.
The rest is styling because of ctrl+k ctrl+d.
It's a bit more manageable if you hide whitespace.

Reply:
Can you revert the changes? This a generated file and we should not re-style it. The changes will be lost next time the file is generated.

Reply:
Check my other comment as well